### PR TITLE
Mobile responsive Milestone

### PR DESCRIPTION
**Pull request by MxoDevops**
##Portfolio updates

In this portfolio I ;
-Added the background image according to the Figma specification
-Added google fonts
-Made the portfolio mobile responsive

### CODE REVIEW

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @MxoDevops , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      <nav class="nav-bar">
+        <a class="container"><b>Mx</b></a>
+        <ul class="menu">
+          <li class="item"><a href="#">Portfolio</a></li>
+          <li class="item"><a href="#">About</a></li>
+          <li class="item"><a href="#">Contact</a></li>
+          <li class="item"><i class="fa fa-envelope"></i></li>
+        </ul>
+        <div class="toggle"><i class="fa fa-bars"></i></div>
+      </nav>
```
[the remark this answers is not in the document]
Comment (MxoDevops) [reply index.html:25]: im sorry but the logo belongs in the nav bar.

[the remark this answers is not in the document]
Comment (ShahierNashaat) [reply index.html:25]: Hi @MxoDevops It is ok if you want to leave it like that but for the best practise you should put only the minute items in the `nav` tag.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    border-top: 2px solid #fff;
+    margin: 24px;
+  }
+
+  .section {
+    background-image: url("images/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: bottom 90px right;
+    background-size: 14vh;
+  }
```
Comment (phelian23) [inline style.css:172]: - [ ] Good job working on the requested change, however, the mobile page background image is too small, kindly make it look like that of the Figma template as much as possible
Your design
Capture

Figma template
figma

Comment (MxoDevops) [reply style.css:172]: I'm sorry but did you use the recommended mobile dimension as specified on Figma?

Comment (phelian23) [reply style.css]: Yes, however, your page should be responsive irrespective of the dimension used. Kindly make sure it is

**Review by m05e5 — APPROVED** [review]
# Hi @MxoDevops , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**